elytron/src/main/resources/schema/wildfly-elytron_8_0.xsd

```diff
@@ -1008,6 +1008,14 @@
                         </xs:documentation>
                     </xs:annotation>
                 </xs:attribute>
+                <xs:attribute name="principal-transformer" type="xs:string">
```

Comment:
```use="optional"``` could be added here (this is the default anyway but helps for readability).

Reply:
added